website/src/pages/index.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from "react";
+import Layout from "@theme/Layout";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import classnames from "classnames";
+import withBaseUrl from "@docusaurus/withBaseUrl";
+
+const features = [
+	{
+		title: "Optimizing",
+		content:
+			"Redex provides a framework for reading, writing, and analyzing .dex files, and a set of optimization passes that use this framework to improve the bytecode. An APK optimized by Redex should be smaller and faster."
+	},
+	{
+		title: "Fast",
+		content:
+			"Fewer bytes also means faster download times, faster install times, and lower data usage for cell users. Lastly, less bytecode also typically translates into faster runtime performance."
+	},
+	{
+		title: "Buck Integration",
+		content:
+			"Redex has deep integration with Buck where your Redex config is passed as a parameter to the Buck android_binary rule when generating the APK."
+	}
+];
+
+class Button extends React.Component {
```

Comment:
Don't need a Button component. Just do

```jsx
import Link from '@docusaurus/Link`;

...

<Link className="button button--outline" href={withBaseUrl('...')} />
```

Using our `<Link>` component will cause it to do a front end redirect instead of full-page redirect.